workbench-core/user-management/src/plugins/cognitoUserManagementPlugin.ts

```diff
@@ -421,7 +436,13 @@ export class CognitoUserManagementPlugin implements UserManagementPlugin {
         })
       );
 
-      return users.filter((user) => user.id);
+      const data = users.filter((user) => user.id);
```

Comment:
What does this filter method do? Is it returning all users that have an `id`? Shouldn't all users have an `id`?

Reply:
It may be for type safe-ness. This was MAF code so I don't have the full context but there are tests to cover not returning users without IDs